pandas/io/formats/terminal.py

```diff
@@ -94,21 +94,26 @@ def _get_terminal_size_tput():
     # get terminal width
     # src: http://stackoverflow.com/questions/263890/how-do-i-find-the-width
     # -height-of-a-terminal-window
+    output_cols = None
+    output_rows = None
     try:
         import subprocess
         proc = subprocess.Popen(["tput", "cols"],
```

Comment:
can you add a 1-liner comment on what is going on here

Reply:
But you said these aren't necessary... should I remove them, or add a comment explaining why I declared these variables before the try/except block?

Reply:
remove these. ahh there is already a comment, so ignore that.